flocker/node/script.py

```diff
@@ -294,7 +365,7 @@ class AgentServiceFactory(PClass):
     # This should have an explicit interface:
     # https://clusterhq.atlassian.net/browse/FLOC-1929
     deployer_factory = field(mandatory=True)
-    get_external_ip = field(initial=_get_external_ip, mandatory=True)
```

Comment:
I'm not moving to a vendored pyrsistent, just a forked one.
Vendoring is a longer-term thing.